sections/tooling/stylelint.md

```diff
@@ -173,7 +173,7 @@ The full list of supported tags:
 - `sc-property`
 - `sc-value`
 
-> If you are in doubt of the vocabulary you can refer to [this CSS vocabulary list](http://apps.workflower.fi/vocabs/css/en) with examples.
+> If you are in doubt of the vocabulary you can refer to [this CSS vocabulary list](https://byui-cit.github.io/wdd130/activities/cssvocab.html) with examples.
```

Comment:
Let's use this link to MDN https://developer.mozilla.org/en-US/docs/Web/CSS/Reference